x/staking/client/cli/flags.go

```diff
@@ -30,9 +31,6 @@ const (
 	FlagGenesisFormat = "genesis-format"
 	FlagNodeID        = "node-id"
 	FlagIP            = "ip"
-
```

Comment:
now  they are defined  under general flags  because they're also used in `gentx`, `sim` etc